src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+html {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  height: 60px;
+  position: relative;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  position: absolute;
+  left: 50px;
+  top: 10px;
+}
+
+.nav {
+  position: absolute;
+  right: 50px;
```

Comment:
You don't need to use `position: absolute;` here, implement it differently

Reply:
Just use the usual flow of these elements, supplementing them with paddings/margins